### ISSUE TITLE
Auto-login issues

### ISSUE THREAD

**Issue by entropy-13** [TheCheatsrichter/Gw2_Launchbuddy#407]
First, thank you for all the hard work developing and updating the software!!

On build 3.2.4 - Auto-login is still bugged. Appears to hang at 80% and then replaces the saved password with 2 characters (**)

**To Reproduce**
Removed all *.dlls from GW2 folder.  Deleted loginfile. Saved. Restarted Launchbuddy and recreated loginfile. Saved. Tested: Auto-login works. 
Add latest d3d11.dll to GW2 folder. Add dll via injected software. Auto-login fails.
Clear dll from folder. Reset loginfile and verify it works. Add d3d11.dll to GW2 folder. DO NOT add dll via injected software. Auto-login fails.

**Expected behavior**
Previous builds were compatible with d3d11.dll AND other dlls related to arcdps.

**First happend in LB Version**
Issue first appeared in LB Version : 3.2.2

**Screenshots**

**Comment by entropy-13** [comment]
Finally had a chance to read through some of the other posts about this issue.  Moving the DLLs to a separate folder and injecting them allows the auto-login function to work correctly.

**Comment by brybrant** [comment]
TLDR: I think launchbuddy should detect and warn the user if arcdps+addons are in the Guild Wars 2 folder.

I was having this same issue for a long time; i created a fresh loginfile but when I clicked launch, the launcher username field is empty! I tried to manually type my username but launchbuddy is spamming enter while I'm typing so the login attempt fails and then I have to enter both username and password, making launchbuddy redundant.

I thought launchbuddy was failing to save my loginfiles correctly, so I reinstalled, checked folder permissions, firewall, I tried everything to fix it. I was about to give up and uninstall launchbuddy until I saw this and learned you cannot load arcdps+addons normally by putting the dll files into the Guild Wars 2 directory as per arcdps install instructions - instead you must move the dll files to a separate folder and inject them in launchbuddy account settings.

Now launchbuddy works perfectly but I wish it detected my addons and warned me to inject them instead! That would have saved me a lot of stress.
